.kitchen.yml

```diff
@@ -54,19 +51,30 @@ suites:
         server_repl_password: q654321
```

Comment:
Hmm, my plan with this was to patch the postgresql support to suit our needs and not touch the mysql support. I could remove it if you'd rather? To be clear, this is a compatibility patch to an upstream application cookbook, we'll have a wrapper cookbook for this as well with our own config in.